locales/en/base.json

```diff
@@ -1,33 +1,84 @@
 {
```

Comment:
Of course we also need to transform the other languages when this is decided 😊

Reply:
Ofc